#Request
int32 bytesToRead
int32 pollRate
---
#Response
int32 response
